aio/src/styles/0-base/_typography.scss

```
@@ -6,59 +6,73 @@ body {
   font-family: $main-font;
   margin: 0;
   color: $darkgray;
-  @include font-size(14);
+  @include font-size(16);
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
 h1, h2, h3, h4, h5, h6 {
   color: $deepgray;
+  margin-bottom: 0;
+  font-weight: 500;
+  position: relative;
 }
 
 h1 {
-  @include font-size(24);
-  font-weight: 500;
+  @include font-size(40);
   display: inline-block;
-  margin: 8px 0px;
+  margin: 1.6rem 0;
 
   @media screen and (max-width: 600px) {
     margin-top: 0;
   }
 }
 
 h2 {
-  @include font-size(22);
-  font-weight: 500;
-  margin: 32px 0px 24px;
+  @include font-size(32);
+  margin-top: 2rem;
   clear: both;
+
+  &:before {
+    content: "";
+    display: block;
+    border-bottom: 1px solid $lightgray;
+    margin: 4rem 0;
+  }
+}
+
+aio-shell {
+  &.page-resources, &.page-events, &.page-features, &.page-presskit, &.page-contribute, &.page-about, &.page-api {
+    main h2:before {
+      display: none;
+      border: 0;
+      visibility: hidden;
+    }
+  }
```

I think it is better nest the special styles inside the `h2:before` block:

```suggestion

    .page-resources &, .page-events &, .page-features &, .page-presskit &, .page-contribute &, .page-about &, .page-api & {
      display: none;
      border: 0;
      visibility: hidden;
    }
  }
```